### PR TITLE
Fix/pages and routes

### PR DESCRIPTION
**Pull request by johnflank**

### CODE REVIEW

File: v2/src/components/SiteHeader/index.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default function SiteHeader({ theme, ...props }) {
           <>
             <div className="max-w-6xl mx-auto px-4 sm:px-6 flex justify-between items-center py-3 md:space-x-10">
               <div className="flex justify-start">
-                <Link to={routes.home.path}>
+                <a href={routes.home.path}>
```
Comment (flanksaul) [inline v2/src/components/SiteHeader/index.js:93]: This probably shouldn't be an <a> tag.

File: v2/src/components/SiteHeader/index.js [hunk 2]
```diff
@@ -119,13 +120,13 @@ export default function SiteHeader({ theme, ...props }) {
               <Popover.Group as="nav" className="hidden md:flex space-x-10">
                 {navbarItems.map((item) => {
                   return (
-                    <Link
+                    <a
```
Comment (flanksaul) [inline v2/src/components/SiteHeader/index.js:123]: I don't think this should be an <a> tag

File: v2/src/pages/About/index.js [hunk 3]
```diff
@@ -118,10 +118,9 @@ const features = [
 
 export default function About() {
   return (
-    <DefaultLayout className="bg-gray-50" title="About Us">
-
+    <DefaultLayout  title="About Us">
```
Comment (flanksaul) [inline v2/src/pages/About/index.js:121]: There's a library called react-helmet which I think should be used to set things like titles.

**Comment by moshloop** [comment]
https://github.com/lorin/resilience-engineering

File: v2/src/pages/Home/index.js [hunk 4]
```diff
@@ -56,11 +56,7 @@ export default function Home() {
           imageSrc={resilienceReviewImage}
           actions={
             <div className="flex flex-col items-start md:flex-row space-y-4 md:space-y-0 space-y-0 md:space-x-4">
-              <LinkButton
-                to="/" // @TODO: Change to a correct link
-              >
-                Learn More
-              </LinkButton>
+              <LinkButton href={routes.services.path}>Learn More</LinkButton>
```
Comment (flanksaul) [inline v2/src/pages/Home/index.js:59]: From what I can tell, this will produce an <a> tag.

Comment (flanksaul) [reply v2/src/pages/Home/index.js:59]: Same comment for all instances.

File: v2/src/components/SiteFooter/index.js [hunk 5]
```diff
@@ -93,7 +93,8 @@ export default function SiteFooter() {
       links: [
         { name: "About", href: `${routes.about.path}` },
         { name: "Blog", href: "/blog" },
-        { name: "Careers", href: "/careers" }
+        { name: "Careers", href: "/careers" },
```
Comment (flanksaul) [inline v2/src/components/SiteFooter/index.js:96]: Can we use the router paths consistently here?

**Pull request merged**